ganga/GangaCore/Lib/Localhost/LocalHostExec.py.template

```diff
@@ -85,6 +88,26 @@ errorfile=open('stderr','w')
 sys.stdout=open('./__syslog__','w')
 sys.stderr=sys.stdout
 
+if virtualization:
+    from Virtualization import checkDocker, checkUDocker, checkSingularity, installUdocker
+
+if (virtualization == "Docker"):
+    appscriptpath[0] = "/work_dir/"+appscriptpath[0]
+    if (checkDocker()):
+        print("Using Docker")
+        appscriptpath = ['docker', 'run', '-v', workdir+":"+"/work_dir", virtualization_image] + appscriptpath        
+    else:
+        print("Docker not availabe or user doesn't have the permission to run docker demon")
+        if not (checkUDocker()):
+            installUdocker()
+        print("Using UDocker")
+        fullenvironment["PROOT_NO_SECCOMP"]="1"
```

Comment:
The `fullenvironment` is just a dictionary. This will not change anything for the `udocker` script.